spec/graphql/mutations/add_barn_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+module Mutations
+  RSpec.describe AddBarn, type: :request do
```

Comment:
I haven't seen this pattern before. I usually see something like this. If this works for you I don't think it's a big deal.

```rb
RSpec.describe Mutations::AddBarn, type: :request do
```